browser/ui/webui/brave_blank_page_ui.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_WEBUI_BRAVE_BLANK_PAGE_UI_H_
+#define BRAVE_BROWSER_UI_WEBUI_BRAVE_BLANK_PAGE_UI_H_
+
+#include <string>
+
+#include "content/public/browser/web_ui_controller.h"
+
+class BraveBlankPageUI : public content::WebUIController {
+ public:
+  BraveBlankPageUI(content::WebUI* web_ui,
+                   const std::string& host);
+  ~BraveBlankPageUI() override;
+
+  BraveBlankPageUI(const BraveBlankPageUI&) = delete;
+  BraveBlankPageUI& operator=(BraveBlankPageUI&) = delete;
+};
```

Comment:
Instead of creating a whole new WebUI route and class, I was thinking we could just use the regular `chrome://newtab` route and class, and simply change the .html resource that is rendered, depending on the value of `kNewTabPageShowsOptions`. I can't think of a reason that we would need to have separate routes or both the dashboard route and the blank route available at the same time, regardless of the preference value. Do you have a different opinion?

Reply:
I thought new tab and blank are different webui. So, I created new route.
But, I don't have strong opinon about this. I'll follow your suggestion.

Reply:
@petemill  Fixed. PTAL again!

Reply:
I realized that your approach is better because blank page can also act like NTP. - search suggestion in location bar.